### ISSUE TITLE
[exporter/prometheusremotewrite] Support for delta Exponential Histograms

### ISSUE THREAD

**Issue by mx-psi** [open-telemetry/opentelemetry-collector-contrib#22806]
### Component(s)

exporter/prometheusremotewrite

### Is your feature request related to a problem? Please describe.

This issue is similar to #19153; currently the Prometheus Remote Write exporter only supports cumulative exponential histograms and drops delta exponential histograms (see #17370 and the related spec guidance added on open-telemetry/opentelemetry-specification/pull/3079).

### Describe the solution you'd like

Delta exponential histograms are accumulated into a cumulative exponential histogram and then the cumulative exponential histogram to Prometheus native histogram conversion logic is applied. To merge the exponential histograms, the lowest scale among them should be used.

### Describe alternatives you've considered

We could have this functionality on a separate processor. The proposed solution for #19153 seems to add functionality in the exporter itself, so I went with that, but I also see advantages on having a separate processor for this.

### Additional context

This needs a change in the specification, since it currently says:

> OpenTelemetry Exponential Histogram metrics with the delta aggregation temporality are dropped.

It would have to be changed to say

> OpenTelemetry Exponential Histograms with Delta aggregation temporality SHOULD be aggregated into a Cumulative aggregation temporality and follow the logic above, or MUST be dropped.

similar to what we say for OpenTelemetry Histograms with Delta aggregation temporality.

---

This issue is specific to the PRW exporter since the Prometheus exporter does not support exponential histograms yet, but I tagged it with `exporter/prometheus` since we should take this exporter also into account.

**Comment by MovieStoreGuy** [comment]
@mx-psi do you know if anyone is interested on working on this?

**Comment by mx-psi** [comment]
> @mx-psi do you know if anyone is interested on working on this?

No, I opened this on behalf of a customer, mostly to gauge interest and see if other folks were interested. I am not planning to work on this at the moment

**Comment by jakegut** [comment]
Rather than converting from delta to cumulative would it be possible to send them as gauge histograms instead?

Looking at the original design doc and the protobuf spec it seems like gauge histograms are just delta histograms.
